lingua_franca/lang/parse_de.py

```diff
@@ -143,20 +144,28 @@ def repl(match):
     return (duration, text)
 
 
-def extract_number_de(text, short_scale=True, ordinals=False):
+def extract_number_de(text, short_scale=True, ordinals=False, decimal='.'):
```

Comment:
the default of `decimal` should be a comma, as this is the common form

Reply:
this is part of https://github.com/OpenVoiceOS/ovos-lingua-franca/pull/43 which was dragged in this PR accidentally